core/common/base.py

```diff
@@ -74,207 +75,47 @@ def receive_data(self, size: int = 4096, data_type: str = str) -> Optional[Any]:
 
 class Server(BaseSocketServer):
     """
-    A class used to abstract the socket that will act as the server
+    A class used to abstract the socket that will act as the server.
 
     ....
     """
 
     def __init__(self, ip, port):
-        """
+        """Initialize Server Object with ip and port parameters.
+
         Parameters
         ----------
-
         ip : str
             A string specifying the ip to bind the socket with.
 
         port: int
             A number specifying the port to bind the socket with.
         """
-
-        super(Server, self).__int__(socket.socket(socket.AF_INET, socket.SOCK_STREAM))
+        super(Server, self).__init__(socket.socket(
+            socket.AF_INET, socket.SOCK_STREAM))
         self.socket.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
         self.socket.bind((ip, port))
         self.socket.listen(10)
 
 
 class Client(BaseSocketServer):
     """
-    A class used to abstract the socket that will connect with the server
+    A class used to abstract the socket that will connect with the server.
 
     ...
     """
 
     def __init__(self, ip, port):
-        """
+        """Initialize Client Object with ip and port parameters.
```

Comment:
Codacy found an issue: [Multi-line docstring summary should start at the second line (D213)](https://app.codacy.com/manual/excalibur.krv/m3u8-dl/pullRequest?prid=6274429)